public/src/components/channelManagement/headerTests/utils/defaults.ts

```diff
@@ -0,0 +1,66 @@
+import { Cta, UserCohort } from '../../helpers/shared';
+import { HeaderTest, HeaderVariant } from '../../../../models/header';
+
+import { getStage } from '../../../../utils/stage';
+
+export const DEFAULT_PRIMARY_CTA: Cta = {
+  text: 'Support the Guardian',
+  baseUrl: 'https://support.theguardian.com/contribute',
+};
+
+export const DEFAULT_SECONDARY_CTA: Cta = {
+  text: 'Support the Guardian',
+  baseUrl: 'https://support.theguardian.com/contribute',
+};
+
+const DEV_AND_CODE_DEFAULT_VARIANT: HeaderVariant = {
+  name: 'CONTROL',
+  content: {
+    heading: 'Lorem Ipsum',
+    subheading: 'Exam desut lineas buteram loas',
```

Comment:
We use realistic copy for default values in all environments, just in case...
Let's go with:
```
heading: 'Support the Guardian',
subheading: 'Available for everyone, funded by readers',
```

Reply:
Updated both variants where LoremIpsum copy was being used